edu.ohio_state.khatchad.refactoring/src/edu/ohio_state/khatchad/refactoring/ConvertConstantsToEnumRefactoring.java

```diff
@@ -89,6 +89,7 @@
 
 public class ConvertConstantsToEnumRefactoring extends Refactoring {
 
+	//what is this SearchMatchPurose class doing
```

Comment:
Please remove this comment and direct questions to the [mailing list](convert-constants-to-enum-dev@googlegroups.com).